src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -132,6 +133,20 @@ class UiaTextRangeTests
         short yPos;
     };
 
+    enum SupportedTextAttributes
+    {
+        FontName = UIA_FontNameAttributeId, // special handling
+        FontSize = UIA_FontSizeAttributeId, // special handling
+        FontWeight = UIA_FontWeightAttributeId,
+        ForegroundColor = UIA_ForegroundColorAttributeId,
+        Italic = UIA_IsItalicAttributeId,
+        IsReadOnly = UIA_IsReadOnlyAttributeId, // special handling (always false)
+        StrikethroughColor = UIA_StrikethroughColorAttributeId,
+        StrikethroughStyle = UIA_StrikethroughStyleAttributeId,
+        UnderlineColor = UIA_UnderlineColorAttributeId,
+        UnderlineStyle = UIA_UnderlineStyleAttributeId

```

Comment:
Why are we redefining them with shorter names?

Reply:
I was lazy and hoped it would make the code more legible... it didn't haha. Removed.